dags/fivetran_acoustic.py

```diff
@@ -199,15 +198,9 @@ def _generate_acoustic_report(conn_id: str, report_type: str, config: Dict[Any,
             poke_interval=30
         )
 
-        downstream_dag = dag_id.replace("fivetran", "bqetl")
-
-        trigger_dependent_dag_and_wait = TriggerDagRunOperator(
-            task_id="trigger_dependent_bqetl_process",
-            trigger_dag_id=downstream_dag,
-            wait_for_completion=True,
-            execution_date="{{ ds }}",
-            reset_dag_run=True,
+        load_completed = DummyOperator(
+            task_id='fivetran_load_completed',
         )
 
-        generate_report >> sync_trigger >> sync_wait >> trigger_dependent_dag_and_wait
+        generate_report >> sync_trigger >> sync_wait >> load_completed
```

Comment:
Nit: could you point to `sync_wait`/`wait_for_fivetran_connector_completion` directly in `ExternalTaskCompletedSensor`'s? Not sure I follow the need for the dummy operator